debug/debug.go

```diff
@@ -0,0 +1,42 @@
+package debug
+
+import (
+	"fmt"
+	"strings"
+)
+
+var (
+	printCommands    = false
+	verbose          = false
+	packageGenerated = false
+)
+
+func PrintCommand(command string, args ...string) {
+	if !printCommands {
+		return
+	}
+	fmt.Printf("%s %s\n", command, strings.Join(args, " "))
+}
+
+func Log(format string, args ...interface{}) {
+	if !verbose {
+		return
+	}
+	fmt.Printf(format, args...)
+}
+
+func PackageGenerated(pkgPath string) {
+	if !packageGenerated {
+		return
+	}
+	fmt.Println(pkgPath)
+}
+
+func SetPrintCommands(value bool) {
+	printCommands = value
+}
+
+func SetPackageGenerated(value bool) {
```

Comment:
Any reason not to export the variables and remove the setters? We're the only users, so it's not like we need to protect anything.

Reply:
Nope, just my personal preference. Changes to globals.